tools/remove_dependencies.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+
+# Copyright 2019 The Vitess Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Remove tools and dependencies installed by "make tools"
+
+get_arch() {
+  uname -m
+}
+
+function fail() {
+	echo "ERROR: ${1}"
+	exit 1
+}
+
+BUILD_JAVA=${BUILD_JAVA:-1}
+BUILD_CONSUL=${BUILD_CONSUL:-1}
+VTROOT="$PWD/../"
+
+[[ "$(dirname "$0")" = "." ]] || fail "remove_dependencies.sh must be run from its current directory"
+
+uninstall_protoc() {
+    echo "Removing protoc..."
+    local dist="$1"
+
+    if [ -f $dist ]; then
+        unlink "$dist/bin/protoc"
+        rm "$VTROOT/bin/protoc"
+    fi
+    rm -rf $dist
```

Comment:
I don't really understand this. If the passed path is a file, why are we then treating it like a directory?

Reply:
Yeah, that was a mistake, it should be `-f "$dist/bin/protoc"` instead.